routing/road_graph.cpp

```diff
@@ -59,39 +58,35 @@ string DebugPrint(Junction const & r)
 
 // Edge ------------------------------------------------------------------------
 // static
-Edge Edge::MakeFake(Junction const & startJunction, Junction const & endJunction, bool partOfReal)
+Edge Edge::MakeFake(Junction const & startJunction, Junction const & endJunction)
 {
-  return MakeFakeWithForward(startJunction, endJunction, partOfReal, true /* forward */);
+  Edge e{FeatureID(), true /* forward */, 0 /* segId */, startJunction, endJunction};
+  e.m_type = 0;
```

Comment:
Could you please use named constants?

Reply:
Either rename to m_typeMask, either use named constant.

Reply:
>Either rename to m_typeMask, either use named constant.
+1